### PR TITLE
add dep and con pars tasks

### PR DESCRIPTION
**Pull request by jonborchardt**
this uses hierplane for now

### CODE REVIEW

File: ui/src/@types/hierplane.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * hierplane does not have types
+ */
+declare module 'hierplane' {
+    const Tree: (spec: any) => any;
```
Comment (codeviking) [inline ui/src/@types/hierplane.d.ts:5]: We should be able to at least type this as a `React.PureComponent` or something like that.  For the props I'd make `tree` a `{}`.

It's better than the `any` route, and shouldn't be too much work.

File: ui/src/components/Hierplane.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    .hierplane__visualization-verbs__prev {
+        transform: rotateY(-180deg);
+    }
+
+    .hierplane__visualization-verbs__label {
+        user-select: none;
+    }
+
+    /* THESE ARE HACKS PUT HERE BY CODEVIKING. */
```
Comment (codeviking) [inline ui/src/components/Hierplane.tsx:45]:
```suggestion
    /* THESE ARE HACKS PUT HERE BY @codeviking */
```

File: ui/src/demos/constituency-parsing/Predictions.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export const Predictions = ({ input, model, output }: Props) => {
+    // TODO: this is a placeholder visualization. We will be removing Hierplane someday soon!
+    return (
+        <Output.Section>
+            {output.hierplane_tree ? (
+                <Output.SubSection>
+                    <Hierplane tree={output.hierplane_tree} theme="light" />
+                </Output.SubSection>
+            ) : null}
```
Comment (codeviking) [inline ui/src/demos/constituency-parsing/Predictions.tsx:22]: Should we at least print the output when the tree isn't returned, maybe a `<code>{JSON.stringify(output, null, 2)}</code>`.  Maybe that's already taken care of by the debug output, in which case we could just let the user know to look there: `Unable to render output. See debug output below`.

**Review by codeviking — APPROVED** [review]
Nice, moving right along!

File: ui/src/@types/hierplane.d.ts (hunk 1 above)
Comment (jonborchardt) [reply ui/src/@types/hierplane.d.ts]: done

File: ui/src/demos/constituency-parsing/Predictions.tsx (hunk 3 above)
Comment (jonborchardt) [reply ui/src/demos/constituency-parsing/Predictions.tsx]: added

**Pull request merged**